web/api/users.js

```diff
@@ -318,6 +348,13 @@ exports.register = function (server, eOptions, next) {
           ],
         },
       },
+      validate: {
+        payload: {
+          token: Joi.string().guid(),
```

Comment:
These should all be `.required()`, right?

Reply:
yay should :+1: